docs/snippets/start-connect-dydx-snippet.mdx

```diff
@@ -1,49 +1,74 @@
-The dYdX blockchain requires some additional steps to start Connect.
+**The minimum required version for Connect with dYdX is `v1.0.5+`.**
 
-**The minimum required version for Connect with dYdX is `v1.0.1+`.**
-
-First, please ensure you've received your API keys for Raydium markets. If you have not received API keys, please reach out to the team in the relevant channels.
+First, please ensure you've received your **API keys** for Raydium markets. If you have not received API keys, please reach out to the team in the relevant channels.
 
 Next, place your API keys under their corresponding URLs in the following file and save it to your system. Keep the file path handy as we will pass it into a flag when running Connect.
+
+We will also supply an edited configuration for the `dydx_migration_api` which facilitates graceful migration from dydx's `x/prices` to `x/marketmap`.
+
+**Make sure to fill in the URL for the REST endpoint and gRPC endpoint of your node** (in that order). The migration API will _not_ work if the REST API endpoint is not the first endpoint in the list.
+
 ```json oracle.json
 {
-   "providers": {
-      "raydium_api": {
-         "api": {
-            "endpoints": [
+   "providers":{
+      "dydx_migration_api":{
+         "name":"dydx_migration_api",
+         "api":{
+            "enabled":true,
+            "timeout":20000000000,
+            "interval":10000000000,
+            "reconnectTimeout":2000000000,
+            "maxQueries":1,
+            "atomic":true,
```

Comment:
I kind of prefer the config not having all these fields unless we need to.

Reply:
fixed [remove extra fields](https://github.com/skip-mev/slinky/pull/633/commits/bec7565ec6a359dc48d4525d39846509809d8f44)